File: assignments/se/module-3/q3.c
// WAP to find simple interest

#include<stdio.h>

int main(){
    
    float p , r;
    int t;
    float si;
    printf("Enter value of p:");
    scanf("\n%f",&p);
    printf("Enter value of r:");
    scanf("\n%f",&r);
    printf("Enter value of t:");
    scanf("\n%d",&t);
    si = p*r*t/100;
    printf("%f",si);
    
    
}

File: assignments/python/module-6_fundamentals_of_python/5_loops/lab_questions/q3.py
# q - Practical Example 3: Write a Python program to find a specific string in the list using a simple for loop and if condition. 

# Answer :

fruits = ['apple', 'banana', 'mango']
find = input("enter fruit you want to find :")

for fruit in fruits:
    if fruit == find:
        print(find, "found in list")
        break
else :
    print(find, "sorry can't found ") 
          



